tap_sftp/client.py

```diff
@@ -138,16 +178,33 @@ def get_files(self, prefix, search_pattern, modified_since=None):
         for f in matching_files:
             LOGGER.info("Found file: %s", f['filepath'])
 
+        filtered_files = []
+        for f in matching_files:
```

Comment:
@harshpatel4crest We are traversing the same 'matching_files' list again. See line no. 178